homework_iterators/fizzBuzz.js

```diff
@@ -0,0 +1,34 @@
+const checkFizzBuzz = (n) => {
+  if (!(n % 3)) return 'Fizz';
+  if (!(n % 5)) return 'Buzz';
+  if (!(n % 15)) return 'FizzBuzz';
+  return n;
+};
+
+//! Iterator solution
+
+const arr = [...Array(100).keys()].map((i) => i + 1);
+const arrIter = arr[Symbol.iterator]();
```

Comment:
Do not use the internal iterator of an array - you need to create a new one by your self.

Reply:
remove `arr`. use a pure loop instead.